### DISCUSSION TITLE
Welcome to mono Discussions!

### DISCUSSION THREAD

**Discussion by migueldeicaza** [mono/mono#20655] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iafilm-makers** [comment]
I have just discovered these "Discussions". I have been sharing ideas in issue #9883 in ways that seem to belong here. So a summary and a link.  
  
I am interested in the ability of the Raspberry Pi 4 to run as a mini server using less than 5 watts of power under medium load. I have a windows programming background mostly as a teacher of programming. My education and film-making communities of interest are both interested in energy efficiency in this time of climate change and a good test case is to replace an old PC mini server consuming 40 W at idle, which it does most of the time, with a Raspberry Pi 4 idling at 4 W. The main website of interest and a good immediate demonstration target is an aspx website which just happens to be programmed in VB.NET.  
  
Summary: Success with a trial migration to mono on Pi. My workaround for the VB.NET issue was to migrate the code to C#. More details at: 
https://github.com/mono/mono/issues/9883

EDIT - Migration is complete at: 
https://www.iafilm.co.nz  

EDIT - The Windows version is shut down and is no longer available to compare.

**Comment by miteshsinha007** [comment]
Hi, I can only see mono installs for  LInux , macOS and windows, Any one can redirect me to Solaris and AIX installs please
